File: app/src/main/java/com/stupidtree/hichat/ui/relation/RelationQueryTrigger.java
package com.stupidtree.hichat.ui.relation;

import android.widget.RelativeLayout;

import com.stupidtree.hichat.ui.base.Trigger;

public class RelationQueryTrigger extends Trigger {
    String friendId;
    public static RelationQueryTrigger getActioning(String friendId){
        RelationQueryTrigger rqt = new RelationQueryTrigger();
        rqt.friendId = friendId;
        rqt.setActioning();
        return rqt;
    }

    public String getFriendId() {
        return friendId;
    }
}

File: app/src/main/java/com/stupidtree/cloudliter/ui/relation/RelationQueryTrigger.kt
package com.stupidtree.cloudliter.ui.relation

import com.stupidtree.cloudliter.ui.base.Trigger

class RelationQueryTrigger : Trigger() {
    lateinit var friendId: String

    companion object {
        fun getActioning(friendId: String): RelationQueryTrigger {
            val rqt = RelationQueryTrigger()
            rqt.friendId = friendId
            rqt.setActioning()
            return rqt
        }
    }
}